guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -363,6 +383,43 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
     };
 
     /**
+     * Returns whether the current user can save the user being edited within
```

Comment:
"Returns whether the current user can save the user being edited ..." (etc.)

Meant "clone", yes? Same elsewhere in this comment.